pkg/scheduler/core/generic_scheduler.go

```diff
@@ -802,7 +802,8 @@ func PrioritizeNodes(
 	for i := range nodes {
 		result = append(result, schedulerapi.HostPriority{Host: nodes[i].Name, Score: 0})
 		for j := range priorityConfigs {
-			result[i].Score += results[j][i].Score * priorityConfigs[j].Weight
+			// Expand the range to [0, 100] to be compatible with the range allowed by the framework Score plugins.
+			result[i].Score += results[j][i].Score * priorityConfigs[j].Weight * 10
```

Comment:
On a second thought, can't we just change [MaxPriority](https://github.com/kubernetes/kubernetes/blob/3e962cb61a0787587e5b612940bb1ce539e3eb32/pkg/scheduler/api/types.go#L35) to 100? I am concerned about just multiplying here by 10.

If we do this, then we can set MaxNodeScore = MaxPriority, and so priority and score plugins will be always in sync until priority functions get deprecated.